app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -56,6 +56,11 @@ class LoginFragment : Fragment() {
         setHasOptionsMenu(true)
         showSnackbar()
 
+        val dialog = Dialog(context)
+        dialog.requestWindowFeature(Window.FEATURE_NO_TITLE)
+        dialog.setCancelable(false)
+        dialog.setContentView(R.layout.dialog_progress)
```

Comment:
Can you move dialog logic to utils, so we can display it anytime by passing context?

Reply:
done